de-identification-app/src/test/java/com/ibm/whc/deid/app/endpoint/datamasking/DataMaskingControllerMaskingConfiguration.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * (C) Copyright IBM Corp. 2016,2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.whc.deid.app.endpoint.datamasking;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.startsWith;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.result.MockMvcResultHandlers;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.ibm.whc.deid.app.endpoint.Application;
+import com.ibm.whc.deid.shared.exception.DeidException;
+import com.ibm.whc.deid.shared.pojo.config.ConfigSchemaType;
+import com.ibm.whc.deid.shared.pojo.masking.DataMaskingModel;
+
+@RunWith(SpringRunner.class)
+// force using a test profile to avoid using any other active profile
+// we do not have a real application-test.properties.
+@ActiveProfiles(profiles = { "test" })
+@AutoConfigureMockMvc
+@SpringBootTest(classes = Application.class)
+public class DataMaskingControllerMaskingConfiguration {
```

Comment:
Suggest this class name be DataMaskingControllerMaskingConfigurationTest instead.